addon/services/unified-event-handler.js

```diff
@@ -120,6 +120,7 @@ export default Ember.Service.extend(Ember.Evented, {
         emberEventName,
         targetElement,
         emberHandlers: [],
+        throttledEvent: this._generateThrottledTriggerFn(),
```

Comment:
In general, I'd like to see this call `throttledEventCallback` or something that indicates its a function...

Reply:
Agree, will change.